src/main/scala/DataBaseConn.scala

```diff
@@ -0,0 +1,38 @@
+//DataBaseConnection
+
+object DataBaseConn{
+  import cats.effect.IO
+  import doobie._
+  import doobie.implicits._
+
+  import scala.concurrent.ExecutionContext
+
+  implicit val cs = IO.contextShift(ExecutionContext.global)
+
+  val xa = Transactor.fromDriverManager[IO](
+    "org.postgresql.Driver", "jdbc:postgresql:browserhistory", "postgres", ""
+  )
+
+//  def find(n: String)  =
+//    sql"select id, username, password from users where username = $n"
+//      .query[UserComplete]
+//      .option
+//      .transact(xa)
+//      .unsafeRunSync()
+
+  def getAll: List[UserComplete]  = {
+    val users = sql"select id, username, password from users"
+      .query[UserComplete]
+      .to[List]
+      .transact(xa)
+      .unsafeRunSync()
+    println(users)
+    users
+  }
+
+  def putUser(username: String, password: String) = {
+    sql"insert into users(username, password) values ($username, $password)"
```

Comment:
What would happen here if the username already exists? We should treat this case.

Reply:
I did that in register route: if(getAll.find(i => i.username == userFuture.value.get.get.username).isEmpty) then  putUser

Reply:
I think we can optimize it on a DB level. Having a PK constraint (which also includes a `UNIQUE` clause), we can rely on the fact that trying to insert a new record with the same value for the username will fail, so we could use that.

Also, I think a better name for the method would be `insertUser`/`storeUser`.